packages/scripts/Gruntfile.js

```diff
@@ -0,0 +1,62 @@
+module.exports = function( grunt ) {
+
+	'use strict';
+
+	const pkg = grunt.file.readJSON( 'package.json' );
+	const pluginSlug = grunt.option( 'pluginSlug' ) || pkg.name;
+	const textDomain = grunt.option( 'textDomain' ) || '';
```

Comment:
Should we start a standard of allowing package.json to contain a WordPress object that initially has `textDomain` and `pluginSlug` properties in it?   `pluginSlug` could fallback to `pkg.name` if `pkg.wordpress.pluginslug` isn't defined.

Reply:
I'm more than happy to do it. I thought it might be too early, but if you raised that now let's go for it 💯